socorro/unittest/external/es/test_connection_context.py

```diff
@@ -40,6 +43,12 @@ def test_delete_index(self, es_conn):
         assert index_name not in list(es_conn.get_indices())
 
     def test_delete_old_indices(self, es_conn):
+        # Delete any existing indices first
+        for index in es_conn.get_indices():
+            es_conn.delete_index(index)
+        es_conn.refresh()
+        es_conn.health_check()
```

Comment:
Because I changed `es_conn` to create two indexes, we have to delete them for this test to run.